python/ql/lib/semmle/python/dataflow/new/internal/DataFlowDispatch.qll

```diff
@@ -473,6 +489,7 @@ private TypeTrackingNode functionTracker(TypeTracker t, Function func) {
     result.asExpr() = func.getDefinition().(FunctionExpr).getADecoratorCall()
   )
   or
+  not ignoreForCallGraph(result.getLocation().getFile()) and
   exists(TypeTracker t2 | result = functionTracker(t2, func).track(t2, t))
 }
```

Comment:
(Here and in the other places:)
I would prefer to write it as
```codeql
not ignoreForCallGraph(result.getLocation().getFile()) and (
  ...
)
```
To make it clear that it applies to all disjuncts and should continue to, if the code is updated.

Reply:
I think that's a good idea :+1:

Reply:
I tried this out locally, and it gave absolutely terrible performance, so I've put it off for now.

Reply:
Seriously? That sounds like worth reporting...